eng/templates/stages/secret-validation.yml

```diff
@@ -10,18 +10,15 @@ stages:
   jobs:
   - job: Synchronize
     displayName: Synchronize Secrets
-    pool:
-      name: NetCore1ESPool-Internal-NoMSI
-      demands: ImageOverride -equals 1es-windows-2022
-
+    
```

Comment:
no pool?

Reply:
These lines were overriding the pool for this stage. Now it will use the pool defined in the calling YAML instead

Reply:
ok I see. I thought that we have to implicitly define a pool for each job